ansible/roles/monasca/tasks/config.yml

```diff
@@ -117,6 +117,41 @@
   notify:
     - Restart monasca-log-transformer container
 
+- name: Ensuring logstash patterns folder exists
+  vars:
+    service: "{{ monasca_services['monasca-log-transformer'] }}"
+  file:
+    path: "{{ node_config_directory }}/monasca-log-transformer/logstash_patterns"
+    state: "directory"
+    mode: "0770"
+  become: true
+  when:
+    - inventory_hostname in groups[service['group']]
+    - service.enabled | bool
+
+- name: Find custom logstash patterns
+  local_action:
+    module: find
+    path: "{{ node_custom_config }}/monasca/logstash_patterns"
```

Comment:
Will we only ever need patterns for the transformer?

Reply:
They may be useful for the other logstash services in future. At the moment we are experimenting with them in the log-transformer. Thanks for reviewing.

Reply:
Ok, just thinking there could be a naming issue on localhost since transformer isn't in the path.

Reply:
Will we not need to have different patterns for different logstash services?

Reply:
It seemed more likely to me that you would want to reuse patterns across the logstash services. If there are some patterns specific to a particular service, I don't think that will be a problem so long as you don't have naming collisions. I can use a separate folder for each service if you would still prefer it that way.

Reply:
Happy to defer to you, just wanted to make sure we're not making trouble down the line.